docs/unifiedpush/push-message-format/index.markdown

```diff
@@ -13,27 +13,41 @@ The UnifiedPush Server allows sending messages to the native and non-native Push
        -v -H "Accept: application/json" -H "Content-type: application/json" 
        -X POST
        -d '{
-           "variants" : ["c3f0a94f-48de-4b77-a08e-68114460857e", "444939cd-ae63-4ce1-96a4-de74b77e3737" ....],
-           "alias" : ["user@account.com", "someone@aerogear.org", ....],
-           "categories" : ["someCategory", "otherCategory"],
-           "deviceType" : ["iPad", "AndroidTablet"],
-           "ttl" : 3600,
+           "criteria": {
+             "variants" : ["c3f0a94f-48de-4b77-a08e-68114460857e", "444939cd-ae63-4ce1-96a4-de74b77e3737" ....],
+             "alias" : ["user@account.com", "someone@aerogear.org", ....],
+             "categories" : ["someCategory", "otherCategory"],
+             "deviceType" : ["iPad", "AndroidTablet"]
+           },
            "message": {
              "alert":"HELLO!",
              "sound":"default",
              "badge":7,
              "content-available" : true,
              "action-category" : "some_category",
-             "someKey":"some value",
-             "anotherCustomKey":"some other value"
+             "simple-push": "version=123",
+             "user-data": {
+                "someKey":"some value",
+                "anotherCustomKey":"some other value"
+             },
+             "windows": {                                                
```

Comment:
since the windows bits are only on master, I'd prefer a similar approach that we did with the JAX-RS rest doc:
stable (1.0.x releases) and development (1.1.0-xyz releases):
https://github.com/aerogear/aerogear.org/blob/master/docs/unifiedpush/index.markdown#api-documentation